config/brokers/mt-channel-broker/deployments/broker-ingress.yaml

```diff
@@ -20,6 +22,7 @@ metadata:
   labels:
     eventing.knative.dev/release: devel
 spec:
+  replicas: 0
```

Comment:
ahem... in the "MT broker" ?

I think we should keep the bits for the ST broker ? (at replica 0) 

Or am I missing something here ?

Reply:
Sorry, are you saying that we have this deployment in the config/brokers/mt-channel-broker/ where we set the ChannelBroker deployment to 0?

Reply:
I thought that when we rename the deployment, we should bring the old MT deployments to replicas: 0. Then the one below creates a new deployment?

Reply:
So, you end up with something like this:
note broker-filter, broker-ingress have been brought to 0 and mt-broker-filter and mt-broker-ingress are up and running.

broker-controller is similarly brought to 0.

```
vaikas-a01:eventing vaikas$ kubectl -n knative-eventing get deployments
NAME                   READY   UP-TO-DATE   AVAILABLE   AGE
broker-controller      0/0     0            0           43m
broker-filter          0/0     0            0           34d
broker-ingress         0/0     0            0           34d
eventing-controller    1/1     1            1           34d
eventing-webhook       1/1     1            1           34d
imc-controller         1/1     1            1           34d
imc-dispatcher         1/1     1            1           34d
kafka-ch-controller    1/1     1            1           16d
kafka-webhook          1/1     1            1           16d
mt-broker-controller   1/1     1            1           34d
mt-broker-filter       1/1     1            1           77m
mt-broker-ingress      1/1     1            1           76m
```